config/setup/core.setup.js

```diff
@@ -1,8 +1,9 @@
 import 'jest-styled-components';
```

Comment:
I think that we can remove this line. We are putting the `jest-styled-components` inside the [setupFiles](https://github.com/React95/React95/pull/406/files#diff-0a111a66d5e2d817339a8989d40350dcc6fdb4bd7d5f6fa382a1f3e87e468bc2R20) config, so, I think that this is not necessary.

Reply:
Good catch! This might be a leftover. Removing while typing 🧹